unlock-protocol-com/src/components/interface/Footer/index.tsx

```diff
@@ -13,7 +13,7 @@ const FOOTER_BOTTOM_NAVIGATION: Record<string, LinkType[]> = {
   App: [
     {
       name: 'Creator Dashboard',
-      href: unlockConfig.appURL!,
+      href: `${unlockConfig.appURL!}/locks`,
```

Comment:
See here @searchableguy ^^

Reply:
Yeah. I saw that. I think it might be a little confusing for it to redirected to frontpage by default if you don't add any path.